gbpservice/nfp/service_plugins/vpn/drivers/nfp_vpnaas_driver.py

```diff
@@ -216,3 +258,15 @@ def create_vpnservice(self, context, vpnservice):
             rsrc_id=vpnservice['id'],
             resource=vpnservice,
             reason='create', service_vendor=service_vendor)
+
+    def delete_vpnservice(self, context, vpnservice):
```

Comment:
Why is VPN service delete passed to the configurtor driver? Are we handling this?